contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -347,16 +347,19 @@ def _set_capture_options(self, collect_thread_states: bool, collect_system_memor
         logging.info('Saving "Capture Options"')
         self.find_control('Button', 'OK', parent=capture_options_dialog).click_input()
 
-    def _take_capture(self, length_in_seconds: int):
+    def _take_capture(self, length_in_seconds: int, expect_interrupted_by_service: bool):
         capture_tab = self.find_control('Group', "CaptureTab")
         toggle_capture_button = self.find_control('Button', 'Toggle Capture', parent=capture_tab)
 
         logging.info('Starting to capture for {} seconds'.format(length_in_seconds))
         toggle_capture_button.click_input()
-        time.sleep(length_in_seconds)
```

Comment:
This line got lost in refactoring and should be included in the `else:` block below

Reply:
Whoops.